src/calculateRentalCost.js

```diff
@@ -29,6 +29,23 @@
 
 function calculateRentalCost(days) {
   // write code here
+  let cost = 40 * days;
+  const saleSmall = 20;
+  const saleBig = 50;
+
+  if (days > 6) {
+    cost -= saleBig;
+
+    return cost;
```

Comment:
You can just return the value.
```suggestion
    return cost - saleBig;
```